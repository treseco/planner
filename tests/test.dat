BEGIN:VCALENDAR
BEGIN:VEVENT
SUMMARY:Test Event
DESCRIPTION:TEST
DTSTART:20231004T000000Z
DTEND:20231005T000000Z
END:VEVENT
END:VCALENDAR
