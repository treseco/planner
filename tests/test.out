BEGIN:VCALENDAR
BEGIN:VEVENT
SUMMARY:A Test Calendar Event
DESCRIPTION:TEST
DTSTART:20231204T000000Z
DTEND:20231210T000000Z
END:VEVENT
END:VCALENDAR
